wordpress-mobile/WordPress-iOS
# Filters out a8c p2s from Weekly Roundup candidate sites.

Filters out a8c p2s from Weekly Roundup candidate sites.

@startuptester - This is relevant for testing moving forward.  After this PR we'll no longer get Weekly Roundup notifications in production for A8c p2s.

## Associated PRs:

WPKit: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/445

## To test:

1. Make sure you're on a fresh install - otherwise there may be a delay in the blogs getting synchronized, which is fine, but it may make the testing harder.
2. Go to Me > App Settings > Debug > Weekly Roundup
3. Make sure the toggle "Include a8c p2s" is OFF by default.  This won't be accessible in release builds.
4. Tap "Schedule Immediately".
5. Make sure you get notifications for sites that are not A8c p2s.

Test the same with the toggle ON, and you should get notifications for internal a8c sites too.

For extra security you should try running the same tests by launching the App and changing the date to the following monday after 10 am.  You may need to wait a few hours to see the Weekly Roundup notification.

## Regression Notes

1. Potential unintended areas of impact

Weekly Roundup.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

The testing steps provided above should cover the changes.

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
